configuration/arm64/mysql/add_v2xhub_user.bash

```diff
@@ -7,7 +7,7 @@ arch=$(dpkg --print-architecture)
 if [ $arch = "amd64" ]; then
     REQUIRED_PKG="mysql-client"
 else
-    REQUIRED_PKG="mariadb-client-10.5"
+    REQUIRED_PKG="mariadb-client"
```

Comment:
What does removing version accomplish?

Reply:
@paulbourelly999 This version is not supported in the latest Raspberry Pi OS. Removing allows for the accepted version for the OS to be installed.